tools/helm-docs/readme.gateway-helm.gotmpl

```diff
@@ -18,6 +18,14 @@
 [Helm](https://helm.sh) must be installed to use the charts.
 Please refer to Helm's [documentation](https://helm.sh/docs) to get started.
 
+### Pre-requisites
+
+Install Kubernetes Gateway API CRDs before installing this chart. You can install the CRDs using the following command:
+
+``` shell
+kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/{{ env "GATEWAY_API_VERSION" }}/experimental-install.yaml
```

Comment:
Could we add this file into charts and make it as an option? That would be better for user.